lib/src/services/databases/dataRefresh/implementation.dart

```diff
@@ -0,0 +1,47 @@
+import "package:ramaz/models.dart";
+import "package:ramaz/services.dart";
+import "../../firestore.dart";
+import "interface.dart";
+///
+class CloudDataRefresh implements DataRefreshInterface{
+  ///
+  static final CollectionReference<Map> data = Firestore
+      .instance.collection("dataRefresh");
+
+  @override
+  Future<void> getRefreshData() async{
+    final Map<String, String> map = Map<String,String>.from(await data
```

Comment:
I know I often use generic names like `json` or `data`, but `map` is _too_ generic. You already know it's a map because you can see `Map<String, String>` twice. `json` tells you that you're dealing with a map in JSON form (and you need to process it before using it), and `data` tells you this is coming from the database.